### DISCUSSION TITLE
Quartz Sponge & Golem Pedestal

### DISCUSSION THREAD

**Discussion by InfinityGaben** [Faboslav/friends-and-foes#97] [Ideas]
I feel like tuff golems need a way to be unglued in the event of needing to move them or someone wanting them to move again, maybe a item could be added to do this. I think the item could be a special sponge used to wipe the glue off of the tuff golems, crafting it could be done using a wet sponge with 1 nether quartz. The quartz sponge would be a item that can't be given to tuff golems.

The golem pedestal would be a block that can have a golem bound to it, giving the golem a new home location to sleep at.

**Comment by Faboslav** [comment]
Hello, i did not know this existed, i agree, it is good idea to be able to unglue tuff golem, i will try to come up with something :).

**Comment by Faboslav** [comment]
Tuff Golem can now be unglued with axe.
